### DISCUSSION TITLE
How to enter "Esc" when editing file in vim with libtmux?

### DISCUSSION THREAD

**Discussion by DavdGao** [tmux-python/libtmux#561] [Q&A] [answered]
I'm trying to editing text files in vim with libtmux. In edit mode, I don't know how to pass the `Esc` to the vim in the `send_keys` method. 

Similar problems exist with the other special keys like `Ctrl` and `Alt`

**Comment by tony** [comment, marked as answer]
[`Pane.send_keys()`](https://libtmux.git-pull.com/api/panes.html#libtmux.Pane.send_keys) is a wrapper on top of [`tmux send-keys`](https://man.openbsd.org/OpenBSD-current/man1/tmux.1#send-keys).

I'm not sure about `Esc`, but for `Ctrl` you would do `C`, e.g. `C-a` instead of `Ctrl-a`

`Alt` would be `M`, e.g. `M-a` instead of `Alt-a`.

Aside: Perhaps we can add some docs and additional tests on this.
